smartapps/smartthings/ifttt.src/ifttt.groovy

```diff
@@ -131,19 +131,68 @@ def update() {
 	def type = params.deviceType
 	def data = request.JSON
 	def devices = settings[type]
+	def device = settings[type].find { it.id == params.id }
 	def command = data.command
 
-	log.debug "[PROD] update, params: ${params}, request: ${data}, devices: ${devices*.id}"
-	if (command) {
-		def device = devices?.find { it.id == params.id }
+    log.debug "[PROD] update, params: ${params}, request: ${data}, devices: ${devices*.id}"
```

Comment:
Some funky indentations here and there (and I still dont know if we officially do tabs or spaces)